src/Sylius/Bundle/AdminBundle/Action/ResendOrderConfirmationEmailAction.php

```diff
@@ -31,7 +32,7 @@ final class ResendOrderConfirmationEmailAction
 {
     public function __construct(
         private OrderRepositoryInterface $orderRepository,
-        private OrderEmailManagerInterface $orderEmailManager,
+        private OrderEmailManagerInterface|ResendOrderConfirmationEmailDispatcherInterface $orderEmailManagerOrResendOrderConfirmationDispatcher,
```

Comment:
Due to named arguments mechanism, we shouldn't touch the parameter's name. Also, we should inform it'll be changed in the future.